CICO/views.py

```diff
@@ -87,28 +105,39 @@ def connection(request, formId):
             if form.is_valid():
                 emails = UserCICO.objects.values_list('email')
                 if form.cleaned_data["email"] in emails:
-                    #send mail
+                    # send mail
                     ...
         else:
             form = RequestNewPasswordForm()
 
     return render(request, 'CICO/connexion.html', {"form": form})
 
 
-def profileIndex(request):
+def profileIndex(request, listButton="None"):
+    if listButton == "None":
+        request.session['listStart'] = 0
+    elif listButton == "récent":
+        request.session['listStart'] = max(0, request.session[
+            'listStart'] - LIST_SIZE)  # the max function is used to prevent the substraction from resulting in a negative
+    elif listButton == "ancien":
+        request.session['listStart'] += LIST_SIZE
+
     if not checkIP(request):
+        print("no")
         return render(request, 'CICO/unauthorized.html', status=401)
     items = CiCoItem.objects.all()
     print(request.user)
     print(request.META.get("REMOTE_ADDR"))
     user = request.user
+    recordList = UpdateList(request, UserCICO.objects.get(username=request.user).ownedDevice)
+
     if request.user.is_authenticated:
-        return render(request, 'CICO/profileIndex.html', {"items": items, "user": user.username})
+        return render(request, 'CICO/profileIndex.html',
+                      {"items": items, "user": user.username, "recordList": recordList})
```

Comment:
actually it is since we add `recordList` but I missed it, suggestion, be PEP8 compliant:
```py
return render(request, 'CICO/profileIndex.html', {
    "items": items,
    "user": user.username,
    "recordList": recordList,
})```